app/src/main/java/com/example/playlistmaker/ui/search/AdapterSearchHistory.kt

```diff
@@ -1,16 +1,14 @@
 package com.example.playlistmaker.ui.search
 
-
-import android.content.Intent
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import clickDebounce
 import com.example.playlistmaker.databinding.TrackrecycleviewItemBinding
 import com.example.playlistmaker.domain.models.Track
-import com.example.playlistmaker.ui.player.PlayerActivity
 
-class AdapterSearchHistory :
+
+class AdapterSearchHistory(val callBack: (Track) -> Unit) :
```

Comment:
```suggestion
class AdapterSearchHistory(val onTrackClicked: (Track) -> Unit) :
```
Так будет интуитивно понятно, где именно используется эта лямбда